src/Libraries/CoreNodeModels/Properties/Resources.en-US.resx

```diff
@@ -605,7 +605,7 @@
     <value>equals;tolerance</value>
   </data>
   <data name="EqualsWithToleranceTooltip" xml:space="preserve">
-    <value>Tolerance
+    <value>Tolerance permited for equality check
```

Comment:
NIT: `permitted` should be spelled with double t.

Reply:
Addressed in c07c50d